releasenotes/notes/fixbyteovercount-71db480b4025b60d.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fixes bug on Windows in which NPM displays byte overcounts.
```

Is it correct to say "Windows driver"?

```suggestion
    Fixes a bug on the Windows driver where NPM displays byte overcounts.
```

---

ok... but then how about
`Fixes a bug in the Windows NPM driver where NPM displays byte overcounts.

It reads a bit redundant, but there is more than one driver on windows.